pagopa/io-app
# chore: [IOPID-1372,IOPID-1373] Add remote ToS management

### ⚠️ This PR Depends on https://github.com/pagopa/io-services-metadata/pull/764 ⚠️ ###

## Short description
This PR adds the remote ToS management feature.

<details open><summary>Details</summary>
<p>

| Custom Url | Default Url |
| - | - |
| <video src="https://github.com/pagopa/io-app/assets/16268789/fe048576-f519-4a78-a336-7cb97f0e20d2" />  | <video src="https://github.com/pagopa/io-app/assets/16268789/45b695ba-ed02-4d35-92ae-2a37353e675b" />

</p>

</details> 

## List of changes proposed in this pull request
- Bump io-servives-metadata tag from 1.0.29 to 1.0.30
- Add the remote ToS management

## How to test
Run the app against [this dev server branch](https://github.com/pagopa/io-dev-api-server/tree/IOPID-1374-remote-tos) and try to change [`ToS version`](https://github.com/pagopa/io-dev-api-server/blob/4df1c0c1f17f1e633a1fb6b77cd9df4c7142beff/src/payloads/backend.ts#L125) (with a number greater than the one in the profile) and [`ToS url`](https://github.com/pagopa/io-dev-api-server/blob/4df1c0c1f17f1e633a1fb6b77cd9df4c7142beff/src/payloads/backend.ts#L126) with a custom one. Then restore the ToS url to the expected one `https://io.italia.it/app-content/tos_privacy.html` and test that it's shown too.
